GPII-3199: what to do if production is broken

mrtyler opened the pull request:
Rendered markdown:
https://github.com/mrtyler/gpii-infra/tree/GPII-3199-contact/aws#production-is-broken-theres-been-a-security-breach-or-there-is-some-other-operational-emergency

This is a start to a process that will certainly evolve. For now, the goal is to provide *something* in case we have an operational emergency during the August demos or similar.

This may warrant some real-time discussion. I'll raise it in the next standup.

natarajaya commented:
LGTM.

I think provided emergency reaction plan should be efficient in 99% cases :) I don't mind adding phone numbers, but for me it does not change much –  I always have my phone around, and Slack `@channel` notifications or calls pops out visible/audible alerts. And the only case when I don't have internet on my phone is when I am out of coverage (and unreachable by my phone number as well)

BTW, I am missing in linked "Contact info" document.

natarajaya commented on aws/README.md:
```diff
@@ -134,6 +134,22 @@ Deploying to `prd` requires a [manual action](https://docs.gitlab.com/ce/ci/yaml
+## Production is broken, there's been a security breach, or there is some other operational emergency
+
+If you need to contact a human on the Ops team urgently, here are steps you can try. They escalate in urgency and disruptiveness, so favor earlier steps and patience, but use your best judgement. If production is down right before a big demo or an attack is happening now, it's more important to get an engineer's attention than it is to avoid sending notifications to a few extra people.
+
```
What happened to "don't panic!" point? :)

mrtyler replied:
If this is a serious suggestion, can you propose some encouraging text?

natarajaya replied:
I am pretty bad at encouraging, but I am serious – from my experience, in case of real production failure this helps.

mrtyler replied:
Ok. I'll take a crack at this over in https://github.com/gpii-ops/gpii-infra/pull/75, which happens to contain some other doc fixes I made today.

mrtyler commented:
Sergey, I agree with your general point (many engineers will either see a Slack notification, or they will see nothing) but FYI there are places near me where I have enough signal to receive a text (or maybe a phone call) but not data.

If you are not in the contact info doc, please add yourself :).

I added a phone number column and added mine. I also took another pass over the access list. It is limited to people with RtF accounts, plus a handful of personal email addresses that I recognized as belonging to technical RtF staff. If this seems good to everyone, please add your phone number to the doc.

natarajaya commented:
Ok, added myself to the doc!
